weaviate/collection/data.py

```diff
@@ -356,13 +356,13 @@ def update(
     def get_by_id(
         self, uuid: UUID, metadata: Optional[GetObjectByIdMetadata] = None
     ) -> Optional[_Object]:
-        ret = self._get_by_id(uuid=uuid, metadata=metadata)
+        ret = self._get_by_id(uuid=uuid, includes=metadata)
         if ret is None:
             return ret
         return self._json_to_object(ret)
 
     def get(self, metadata: Optional[GetObjectsMetadata] = None) -> List[_Object]:
-        ret = self._get(metadata=metadata)
+        ret = self._get(includes=metadata)
```

Comment:
change to metadata? same reason as before

Reply:
My feeling here was that `_get` is an internal method that maps the `metadata` user abstraction to the `includes` internal REST protocol. I'll put it one level further down for clarity, however 😁